Warui-K/django-practice
# [Finishes #164067264]Creates set up environment

**What does this PR do?**
Have the set up for Django environment

**Description of Task to be completed?**
Have the following installation set up

Django==2.1.7
pylint==2.3.0
pytest==4.3.0
python-coveralls==2.9.1
pytz==2018.9

- [X] Create a Github Repository, add a README, and clone it to your computer
- [X] Setup the linting library, Style Guide, and Test Framework, and ensure that your work follows the specified style guide requirements.
- [X] Setup the server side of the application using the specified framework.
- [X] Integrate TravisCI for Continuous Integration in your repository (with ReadMe badge).
- [X] Integrate test coverage reporting (e.g. Coveralls) with a badge in the ReadMe.
- [X] Obtain CI badges (e.g. from Code Climate and Coveralls) and add to ReadMe .

**How should this be manually tested?**
After cloning the repo cd into it and ensure you have the setup requirements
Then have the virtual environment activated source venv/bin/activate

**What are the relevant pivotal tracker stories?**
#164067264
